src/data/valid/Biosample-exhasutive.yaml

```diff
@@ -876,8 +875,7 @@ samp_well_name:
   has_raw_value: xxx
 
 sample_link:
-  - IGSN:DSJ0284
-  - any:curie
+  - 'core: soil_core_O'
```

Comment:
I would expect core to be source_mat_id. so i think this one should be removed. Maybe replace core with plant: